whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageService.java

```diff
@@ -110,7 +110,8 @@ private boolean isValidMessage(final EmailMessageInfo message, final List<String
         }
 
         if (!containsAllCaseInsensitive(message.emailAddresses(), outgoingEmail)) {
-            LOGGER.debug("Email {} in outgoing message doesn't match '{}' in failure response", outgoingEmail, StringUtils.join(message.emailAddresses(), ", "));
```

Comment:
Is this change necessary? Previously logged at DEBUG

Reply:
Ah you are correct, debug will not be sent to sentry

Reply:
Actually Sentry does show debug messages , should we change that behaviour ?

Reply:
I see these messages popping in sentry but it must be old  whois tag as nnew one has been changed to debug instead of warn. I will revert the change